lib/tasks/seed_rbac_data.rake

```diff
@@ -0,0 +1,11 @@
+require 'rake'
```

Comment:
@hsong-rh Is this only for testing, since the role seeding would be done by the RBAC service. Even if we need a rake task it would only be for seeding the groups and roles, since the RBAC service currently doesn't provide a way to seed those.

Reply:
Yes, this is only for testing purpose.